app/models/loan_response.rb

```diff
@@ -17,6 +17,7 @@ class LoanResponse
   attr_accessor :end_cell
   attr_accessor :owner
   attr_accessor :breakeven_data
+  attr_accessor :business_canvas_data
```

Comment:
After having gone through the rest of the code I think it would be fine to just replace all `business_canvas_data`s with `business_canvas`